scripts/backend_tests.py

```diff
@@ -54,7 +54,11 @@
     '--test_path',
     help='optional subdirectory path containing the test(s) to run',
     type=str)
-
+_PARSER.add_argument(
+    '-v',
+    '--verbose',
+    help='increase output verbosity',
```

Comment:
I think we should make this a little more detailed, maybe add `by displaying the output of the tests being run`.